code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -1,19 +1,41 @@
 /* Food */
+/datum/reagent/kois_clean
```

Comment:
Why must you hurt me this way? Make this reagent a child of K'ois so you're not literally copy-pasting _all_ of K'ois code. Then either override the affect_x() procs or do as Alberyk suggests and make it dependent on some variable defined in master K'ois code.

Reply:
I was following your code distinction of black k'ois

Reply:
Black K'ois arguably could be a child as well, but the distinction is that it's infection is quite different where as the clean k'ois' infection is literally just an if() check at most.